tracing/ops_tracing/__init__.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2025 Canonical Ltd.
```

Comment:
I used src-layout for Scenario, as it seemed to be the general recommendation for new Python projects. Should we use that here too? `ops` isn't, but is the legacy project in the repo.

Or do you think src-layout was a mistake and maybe we should move Scenario to not use it instead?

Reply:
I don't feel strongly about it...
I went for the concise option, and it just works?
`uv init --package ...` does generate an src layout, while plain `uv init` generates a layoutless app.

I'm not sure about consistency for consistency's sake.